### DISCUSSION TITLE
使用Presets 怎样添加自定义菜单

### DISCUSSION THREAD

**Discussion by anke1460** [dream-num/univer#4260] [Q&A | 常见问题]
如题，使用Presets 怎样添加自定义菜单

**Comment by jikkai** [comment]
两种方法，通过 createUniver 的 plugins 参数，或者返回的 Univer 实例通过 registerPlugin 方法

**Comment by anke1460** [comment]
我需要的是在菜单项里添加一个图标，然后自己设置响应click事件，能否可以用很简便的方式去实现呢，我看自定义UI文档说明给出的例子太繁琐了
